update_git_submodules.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Convenience script for updating git submodules
+# 
+# Requires: git to be installed
+#
+# Usage: ./update_git_submodules.sh from root directory
+
+git submodule foreach git pull origin master
```

Comment:
Reviewers: To confirm, should we pull from origin master or dc master? (after adding dc as a remote)

Reply:
it should be origin. We don't set up "dc" as remote for the submodules.